tests/FSharp.Compiler.ComponentTests/TypeChecks/Graph/FileContentMappingTests.fs

```diff
@@ -122,17 +122,36 @@ module B = C
     | [ TopLevelNamespace "" [ PrefixedIdentifier "C" ] ] -> Assert.Pass()
     | content -> Assert.Fail($"Unexpected content: {content}")
 
-[<Test>]
-let ``Invalid nested module should just be ignored`` () =
-    let content =
-        getContent
-            false
-            """
-module A
 
-module B.C
-"""
+module We_Shouldnt_Crash_With_Invalid_Syntax =
```

Comment:
Nit: I'm not too fond of this snake-cased naming. This is not a common practice in the rest of the codebase. I also dislike the "We shouldn't cash part", go with `module InvalidSyntax` if you insist this should be a nested module.

Reply:
Sure. I think it makes sense to group it in a module so each case doesn't have to explain the common part in the name.